src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/ClusterAwareConnectionProxy.java

```diff
@@ -276,9 +274,10 @@ protected synchronized void initSettings(ConnectionUrl connectionUrl) throws SQL
     this.failoverSocketTimeoutMs = socketTimeout.isExplicitlySet() ? socketTimeout.getValue() : DEFAULT_SOCKET_TIMEOUT_MS;
   }
 
-  protected synchronized void initLogger() {
-    if (!StringUtils.isNullOrEmpty(this.loggerClassNameSetting)) {
-      this.log = LogFactory.getLogger(this.loggerClassNameSetting, Log.LOGGER_INSTANCE_NAME);
+  protected synchronized void initLogger(ConnectionUrl connUrl) {
```

Comment:
I expected just a simple change in PropertyDefinitions that change default logger class.

Reply:
Hmm I think I mentioned that if we change the default logger class to null_logger in property definitions, then logging will be disabled in other parts of the code external to our driver. Since those parts of the code use the standard logger by default right now, would it be best to avoid changing that behavior? Maybe I'm still confused about this though